pkg/operation/botanist/secrets.go

```diff
@@ -487,7 +487,7 @@ func (b *Botanist) SyncShootCredentialsToGarden(ctx context.Context) error {
 				},
 			}
 
-			_, err := controllerutils.CreateOrGetAndStrategicMergePatch(ctx, b.K8sGardenClient.Client(), secretObj, func() error {
+			_, err := controllerutils.GetAndCreateOrStrategicMergePatch(ctx, b.K8sGardenClient.Client(), secretObj, func() error {
```

Comment:
Hmm, I'm not sure if this will work with the SeedAuthorizer. I remember, that `CreateOrGetAndStrategicMergePatch` was chosen for a reason.

Did you check the desired behavior of the quota admission plugin (upstream) on creation requests of existing objects?

Reply:
/hold